### PR TITLE
link to email us

### PR DESCRIPTION
**Pull request by AntonioDaria**
Resolved: https://github.com/Kharouk/CameronCam/issues/35

### CODE REVIEW

**Comment by Kharouk** [comment]
What if we put this into the header rather than have it be under the Login or Register?

**Comment by Kharouk** [comment]
In the "Sign in to report your sightings!" header, we could change it to be "Sign in or email to report your sightings!" and the email part could be clicked on to send an email? @AntonioDaria

File: src/components/Login.js [hunk 1]
```diff
@@ -82,6 +82,13 @@ export default class Login extends Component {
           <button className="buttons register-button" onClick={this.register}>
             Register
           </button>
+          <p>
+            Email us :
```
Comment (Kharouk) [inline src/components/Login.js]: In the "Sign in to report your sightings!" header, we could change it to be "Sign in or email to report your sightings!" and the email part could be clicked on to send an email? @AntonioDaria

Comment (AntonioDaria) [reply src/components/Login.js]: Have a look at the adjustments

**Comment by Kharouk** [comment]
I was thinking something like this @AntonioDaria? This way we don't have extra text and we can link directly to the header?

**Review by AntonioDaria — COMMENTED** [review]
looks good to me

**Review by Kharouk — APPROVED** [review]

**Pull request merged**